epregressions/diffs/thresh_dict.py

```diff
@@ -76,6 +76,9 @@ def lookup(self, hstr):
         if hstr == 'Date/Time' or hstr == 'Time':
             return 0.0, 0.0
 
+        if hstr == 'Version ID':  # allow version number changes to pass without throwing table diffs
```

Comment:
This also sets the threshold for the special Version ID column.  There are a few possibilities here:
1. This change could be removed, since we ignore the diff entirely anyway back in the table_diff routine
2. The change in table_diff could be removed, in which case this threshold would be respected, and it would throw small diffs
3. It is left as-is.  
 
If I have to make any additional changes prior to merge I will probably comment this out but leave it